pywren/wrenhandler.py

```diff
@@ -139,18 +156,18 @@ def aws_lambda_handler(event, context):
     # MKL does not currently play nicely with
     # containers in determining correct # of processors
     custom_handler_env = {'OMP_NUM_THREADS' : '1'}
+
     return generic_handler(event, context_dict, custom_handler_env)
 
 def get_server_info():
 
-    server_info = {'uname' : subprocess.check_output("uname -a", shell=True).decode("ascii")}
+    server_info = {'uname' : " ".join(platform.uname()),
+                   'cpuinfo': platform.processor()}
```

Comment:
do we use a space before `:` or not (the two lines are inconsistent). Also the `cpuinfo` is something we are adding new ?

Reply:
`cpuinfo` replaces `/proc/cpuinfo`